src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +29,42 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+
+        if (null == object) {
+            return false;
+        }
+
+        if (object.getClass().equals(Rectangle.class)) {
+            Rectangle rectangle = (Rectangle) object;
+            return ((null == width && null == rectangle.getWidth())
```

Comment:
```suggestion
            return ((width == rectangle.getWidth())
```
this should be enough